src/common/pool_map.c

```diff
@@ -696,6 +696,16 @@ pool_tree_count(struct pool_domain *tree, struct pool_comp_cntr *cntr)
 	}
 }
 
+int
+pool_map_comp_cnt(struct pool_map *map)
+{
+	struct pool_comp_cntr cntr = {0};
+
+	pool_tree_count(map->po_tree, &cntr);
```

Comment:
pool_buf_extract skips the root when calling pool_tree_count. Does that matter here?

Reply:
oh, this is for pool_buf, instead of pool_map. pool_buf does not have root?

Reply:
I'm not sure... pool_buf_extract and callers of pool_map_comp_cnt appears to have the same goal, thus my reminder above.

Reply:
hmm, in pool_buf_extrac, it allocate pool_buf by cntr.cc_domains + cntr.cc_targets, i.e. no root in pool_buf, which only exits in pool_map.  Or I miss sth?

Reply:
Just discussed with liwei, he is right. I already updated the patch.